Project2/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">  
+    <title>MovieCard</title>
+</head>
+<body style="background-color:#8a2be2"; >
+<div class="card"> 
+<figure>
+<a href="https://www.youtube.com/watch?v=DYYtuKyMtY8"><img class="thumbnail" src="images\SpiderManFFHTrailer-01.png" alt="Spider-Man Far From Home Trailer"></a>
+</figure>
+<div class="words">
+    <h1>Spider-Man:</h1>
+    <h2>Far From Home</h2>
+    <h3>Releasing July 5th, 2019</h3>
+    <a href="https://www.rottentomatoes.com/m/spider_man_far_from_home"><button class="button1">Ratings Details</button></a>
```

Comment:
Here and on lines 32 and 33, you can have **either** a link or a button, but adding both creates a conflict in your code. Since each of these is a link, I recommend just removing the `button` element and styling the link to your liking:
```suggestion
    <a href="https://www.rottentomatoes.com/m/spider_man_far_from_home" class="button1">Ratings Details</a>
```

Note that making this change makes your `.button1` styles have an effect, too: 🙌 
<img width="294" alt="Screen Shot 2019-03-17 at 2 28 16 PM" src="https://user-images.githubusercontent.com/2653980/54496040-eedea480-48c0-11e9-9c78-7a5a592d5a44.png">

What you encountered here is that the `button` element is _heavily_ affected by browser defaults, so it requires _very_ specific CSS to override it. An `a` element is much more flexible and, since it's appropriate here, you're golden 👍 

Also, while I didn't change it above, `button1` isn't a great classname because it's confusing when you add multiple buttons and don't clearly know the difference between `button1` and `button3` or the logic of why they're different or when to use which.